api/tests/Unit/ProtectedRequestUserCheckerTest.php

```diff
@@ -71,10 +77,9 @@ public function testCanUsePrivilegedPermissionProtected()
 
     public function testCanNotUsePrivilegedPermissionUnprotected()
     {
-        $this->markTestSkipped('Enable to test the route protection logic.');
-
         // simulate an unprotected request context
         Request::merge(['isProtectedRequest' => null]);
+        Route::expects('current')->andReturn($this->testRoute);
```

Comment:
Could you add this to every test?  Otherwise, the logic isn't really being tested.

Reply:
This was added in [refactor test to use data provider](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14187/commits/ec8178674526af15f2ffe11aa3f3efaf8482813f)